.internal-ci/charts/reserve-auditor/templates/api-statefulset.yaml

```diff
@@ -0,0 +1,174 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "reserve-auditor.fullname" . }}-api
+  labels:
+    {{- include "reserve-auditor.labels" . | nindent 4 }}
+    app.kubernetes.io/component: api
+spec:
+  replicas: {{ .Values.api.replicas }}
+  selector:
+    matchLabels:
+      {{- include "reserve-auditor.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: api
+  serviceName: {{ include "reserve-auditor.fullname" . }}-api
+  template:
+    metadata:
+      annotations:
+        {{- toYaml .Values.api.podAnnotations | nindent 8 }}
+      labels:
+        {{- include "reserve-auditor.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: api
+    spec:
+      imagePullSecrets:
+        {{- toYaml .Values.imagePullSecrets | nindent 8 }}
+      securityContext: {}
+      volumes:
+      - name: gnosis-config
+        configMap:
+          name: {{ include "reserve-auditor.fullname" . }}-gnosis
+      {{- if not .Values.persistence.enabled }}     
+      - name: data
+        emptyDir: {}
+      {{- end }}
+      containers:
+      {{- with .Values.mobilecoind }}
+      - name: mobilecoind
+        securityContext: {}
+        image: "{{ .image.repository }}:{{ .image.tag | default $.Chart.AppVersion }}"
+        imagePullPolicy: Always
+        ports:
+        - name: mobilecoind-rpc
+          containerPort: 3229
+        livenessProbe:
+          exec:
+            command:
+            - "/usr/local/bin/grpc_health_probe"
+            - "-addr=:3229"
+          failureThreshold: 3
+          periodSeconds: 30
+        readinessProbe:
+          exec:
+            command:
+            - "/usr/local/bin/grpc_health_probe"
+            - "-addr=:3229"
+          failureThreshold: 4
+          periodSeconds: 30
+        volumeMounts:
+        - name: data
+          mountPath: /ledger/
+        - name: data
+          mountPath: /mobilecoind/
```

Comment:
I think you want to make your volumeMounts come from a common point.  You can't dobule-mount the same volume.  You'll want to mount your `data` volume under `/data`, and then have all of your other references be subdirectories off of that, like `/data/auditor`, `/data/mobilecoind`, and `/data/ledger` or something like that.  Otherwise it gets really confusing.  You'll have to audit all of your referenced locations, cli args, etc, for this to work correctly.